ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -13,6 +13,7 @@ spec:
     - name: git_email
       default: "no-reply@redhat.com"
     - name: upstream_repo_name
+      default: ""
```

Comment:
This is just a temporary placeholder value. @amisstea, @wcheang, and I agreed to make it optional, still, it's open to discussion during the meeting for this task.

Reply:
@Allda @MarcinGinszt can you think of a better way to default or generate this value? It's only useful when submit="true" and will probably cause confusion.

Reply:
I don't understand, why it's needed :(